scss/_git_repo.scss

```diff
@@ -1,26 +1,192 @@
+@import "color";
+@import "mixins";
+
 .git-repo-areas {
   margin: auto;
 }
 
 .git-repo {
-  .repo-name {
-    font-size: 1.2rem;
-  }
-  .link-out {
+
+  /* Repo item */
+  .repo-item {
     position: relative;
-    padding: 0.3rem 0.8rem;
-    display: inline-block;
-    vertical-align: top;
-    height: 1rem;
-    &:before {
+    padding: 24px 55px 24px 0;
+    border-bottom: 1px solid $light-grey;
+
+    /* Repo name */
+    & > .name {
+      position: relative;
+      height: 23px;
+      @include text-overflow-ellipsis;
+
+      & > a {
+        display: inline-block;
+        color: $blue;
+        font-size: 20px;
+        line-height: 20px;
+        font-weight: bold;
+        text-decoration: none;
+        cursor: pointer;
+
+        &:hover {
+          text-decoration: underline;
+        }
+      }
+
+      & > .private-tag {
+        display: inline-block;
+        color: $grey;
+        font-size: 12px;
+        line-height: 20px;
+        padding: 0 5px;
+        margin-left: 10px;
+        border: 1px solid $light-grey;
+        border-radius: 3px;
+        vertical-align: top;
+        cursor: default;
+      }
+    }
+    /* End of Repo name */
+
+    /* Repo full name */
+    & > .full-name {
+      height: 17px;
+      margin-top: 10px;
+      color: $grey;
+      font-size: 12px;
+      @include text-overflow-ellipsis;
+    }
+    /* End of Repo full name */
+
+    /* Repo description */
+    & > .description {
+      height: 20px;
+      margin-top: 7px;
+      color: $grey;
+      font-size: 14px;
+      @include text-overflow-ellipsis;
+    }
+    /* End of Repo description */
+
+    /* Repo footer */
+    & > .footer {
+      position: relative;
+      margin-top: 20px;
+      color: $grey;
+      font-size: 13px;
+      @include display-flex;
```

Comment:
why use flex here? 
i think flex is still not supported by most browsers right?